parquet-avro/src/main/java/org/apache/parquet/avro/AvroRecordConverter.java

```diff
@@ -59,6 +62,10 @@
  */
 class AvroRecordConverter<T> extends AvroConverters.AvroGroupConverter {
 
+  private static final String STRINGABLE_PROP = "avro.java.string";
+  private static final String JAVA_CLASS_PROP = "java-class";
+  private static final String JAVA_KEY_CLASS_PROP = "java-key-class";
```

Comment:
Do these three keys come from avro, or can we choose any name we'd like for them? If we can choose the name, they should probably be "parquet.avro.*" and be dot separated right?